src/components/Header/Header.scss

```
@@ -1,6 +1,57 @@
-@import '../../styles/utils/vars.scss';
-@import '../../styles/blocks/logo.scss';
-@import '../../styles/blocks/nav.scss';
+@import '../../Styles/utils/vars';
+
+
+.logo {
+  display: block;
+  width: 80px;
+  height: 26px;
+  padding-left: 16px;
```

You need to add them to the element and not the block element
In this case after you add the img element with the '.logo__image' class you can apply the above properties to that element
.
```suggestion
  width: 80px;
  height: 26px;
  padding-left: 16px;
```